hedyorg/hedy
# 🪲 Revise runtime errors to include actual values

Fixes #5005
Value errors display the actual value that is causing the error instead of the variable name holding this value.

**How to test**
Run Hedy locally and check whether the following programs produce correct error messages. Ensure that you supply text values to all ask commands and that the errors contain the actual value you supplied.
Level 1
```
turn around
```

Level 2
```
play -5
```
```
q is -5
play q
```
```
note is ask what note
play note
```
```
a is ask 'Input'
turn a
```
```
a is ask 'Input'
forward a
```
```
color is ask Color?
color color
```
```
a is ask Provide text here
sleep a
```


Level 6
```
a is ask Provide text here
sleep a
```
```
a is ask 'Provide text here'
print a * 1
```

Level 7
```
a is ask 'Provide text here'
repeat a times print 'Hello folks'
```

Level 12
```
a is ask 'Input'
turn a
```
```
a is ask 'Input'
forward a
```
```
a is ask 'Provide text here'
print a * 1
```
